### PR TITLE
[EBPF-499] Remove "Verification_time" field from the verifier stats output

### PR DESCRIPTION
**Pull request by val06**
### What does this PR do?

Don't include in the output the verification time

### Motivation

non-deterministic value which doesn't reflect the stats about the target programs, but affecting the diff output of the task.

### CODE REVIEW

File: tasks/ebpf.py [hunk 1]
```diff
@@ -160,9 +159,8 @@ def collect_verification_stats(
 
     with open(VERIFIER_STATS) as f:
```
Comment (usamasaqib) [inline tasks/ebpf.py:160]: Do we need to open with write permissions here?

Comment (val06) [reply tasks/ebpf.py:160]: good catch, i "lost" the change while rebasing!

**Review by usamasaqib — APPROVED** [review]

**Pull request merged**